ui/apps/platform/src/utils/dateUtils.ts

```diff
@@ -38,7 +38,7 @@ export function getTime(timestamp: DateLike) {
  * @returns {string} - returns a formatted string for the time
  */
 export function getTimeHoursMinutes(timestamp: DateLike) {
-    return format(timestamp, 'h:mmA');
+    return format(timestamp, 'h:mm A');
```

Comment:
Reminder to get @pedrottimark approval on this change. It was changed to work with the `TimePicker` component as well as align with PatternFly recommendations.

Reply:
Thank you! Approved.